docs/cluster-configuration.md

```
@@ -227,11 +227,12 @@ workload identity.
 **Graduation Status**: Alpha
 
 ### enableMultiArchBootImageImport Feature Gate
-See more details [below](#golden-images-in-heterogeneous-clusters).
+This feature is GA now, and the feature gate is deprecated. It will be removed in a future version.
 
-**Default**: `Disabled`
+Use the `spec.workloadSources.enableMultiArchBootImageImport` field instead.
+See [here](#golden-images-in-heterogeneous-clusters) for more details.
```

markdownlint-cli2 (0.23.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)